packages/hackmode/lib/index.js

```diff
@@ -216,9 +216,7 @@ class N3hHackMode extends AsyncClass {
           return
         case 'publishEntry':
           // Note: opt.data is a EntryData
-          // Bookkeep
-          this._bookkeepAddress(this._publishedEntryBook, opt.data.dnaAddress, opt.data.address)
```

Comment:
Think we're missing out on some efficiencies by just removing this, but totally cool merging for now and @ddd-mtl can take a pass at a later date, possibly just altering the order of operations. Thanks for debugging @Connoropolous !